README.md

```diff
@@ -27,7 +27,7 @@ You can shell into the machine with `vagrant ssh` or `ssh -p 2222 vagrant@localh
   * [Solr 4.10.3](http://lucene.apache.org/solr/)
     * Available at: [http://localhost:8080/solr](http://localhost:8080/solr), for indexing & searching your content.
     * Installed in `/var/lib/tomcat7/solr`
-  * [Apache Karaf](http://karaf.apache.org/)
+  * [Apache Karaf 4.0.1](http://karaf.apache.org/)
```

Comment:
It is still 4.0.1 in fcrepo4-vagrant-base-box 0.1.1 https://github.com/fcrepo4-exts/fcrepo4-vagrant-base-box/blob/cff5219b4476608c9ad155b1903dbf9783a871b4/install_scripts/config#L22